packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.constants.js

```diff
@@ -5,7 +5,10 @@ export const IG_MAP_URL = {
 export const TRAVAUX_URL =
   'http://travaux.ovh.net/?project=20&status=all&perpage=50';
 
+export const GUIDES_URL = 'https://docs.ovh.com/fr/xdsl/';
```

Comment:
```suggestion
export const GUIDES_URL = {
FR: 'https://docs.ovh.com/fr/xdsl/';
}
```
In case more guides has to be added in the future ?